debugtools/DDR_VM/src/com/ibm/j9ddr/corereaders/tdump/zebedee/dumpreader/Dump.java

```diff
@@ -168,19 +168,26 @@ public Dump(String filename, boolean initialize) throws FileNotFoundException, F
             raf = new FileImageInputStream(new RandomAccessFile(filename, "r"));
             fileSize = new File(filename).length();
         } catch (FileNotFoundException e) {
-            if (System.getProperty("os.arch").indexOf("390") == -1) {
+            /*
+             * Note: preprocessor conditions do not apply to internal builds so
```

Comment:
I think this should just be explicit and say IBM Java 8.

Reply:
Sure, I can update the comment.

Reply:
Does this work for you?
```
   * Note: IBM Java 8 builds do not use the preprocessor so the [IF] condition
   * is ignored: This must have the proper control flow ignoring those comments.
```

Reply:
Sure. Pls put it in TDumpReader as well.

Reply:
Done.